shared/dfinity/ic_types.py

```diff
@@ -183,6 +188,20 @@ class AbbrevProposal(TypedDict):
                 '[b3b00ba59c366384e3e0cd53a69457e9053ec987](https:##dashboard.internetcomputer.org/release/b3b00ba59c366384e3e0cd53a69457e9053ec987)\n',
     'title': 'Update subnet 4zbus to replica version b3b00ba5',
     'topic': 'TOPIC_IC_OS_VERSION_DEPLOYMENT'}
+
+    could also be
+
+    {'proposal_id': 102768,
+    'payload': {'version': 'b3b00ba59c366384e3e0cd53a69457e9053ec987',
+                'node_ids': ['4zbus-z2bmt-ilreg-'
+                    'xakz4-6tyre-hsqj4-slb4g-zjwqo-snjcc-iqphi-3qe']},
```

Comment:
```suggestion
                'node_ids': ['4fssn-4vi43-2qufr-hlrfz-hfohd-jgrwc-7l7ok-uatwb-ukau7-lwmoz-tae']},
```
Let's use some node id instead of a subnet id